src/segments/jujutsu.go

```diff
@@ -35,12 +35,25 @@ func (s *JujutsuStatus) add(code byte) {
 }
 
 type Jujutsu struct {
-	Working  *JujutsuStatus
-	ChangeID string
-	ClosestBookmarks string
+	Working          *JujutsuStatus
+	ChangeID         string
+	closestBookmarks string
 	Scm
 }
 
+func (jj *Jujutsu) ClosestBookmarks() string {
+	if jj.closestBookmarks == "" {
```

Comment:
it's cleaner to do an early return instead for readability:

```suggestion
	if jj.closestBookmarks != "" {
	    return jj.closestBookmarks
	}
	
	// actual logic
```